spec/lib/stateoscope/tasks/stateoscope_rake_spec.rb

```diff
@@ -41,5 +41,14 @@
         expect(stateoscope).to have_received(:visualize).with(DummyClass, state_machine_name: 'foo')
       end
     end
+
+    context 'with three arguments' do
+      it 'invokes Stateoscope.visualize with the supplied class and state_machine_name and current_state' do
+        stateoscope = class_double('Stateoscope').as_stubbed_const
+        allow(stateoscope).to receive(:visualize)
+        rake['stateoscope:visualize'].invoke('DummyClass', 'foo', 'bar')
+        expect(stateoscope).to have_received(:visualize).with(DummyClass, state_machine_name: 'foo', current_state: 'bar')
+      end
+    end
```

Comment:
It would be helpful if you could add this variant to the README for documentation purposes.